atmos.yaml

```diff
@@ -40,6 +40,8 @@ components:
     init_run_reconfigure: true
     # Can also be set using 'ATMOS_COMPONENTS_TERRAFORM_AUTO_GENERATE_BACKEND_FILE' ENV var, or '--auto-generate-backend-file' command-line argument
     auto_generate_backend_file: false
+    # Can also be set using 'ATMOS_COMPONENTS_TERRAFORM_APPEND_USER_AGENT' ENV var, or '--append-user-agent' command-line argument
+    append_user_agent: "Atmos/<version> (Cloud Posse; +https://atmos.tools)"
```

Comment:
This defeats the purpose of the default and sets it to a constant. Given that users copy the defaults, this is not desirable outcome.

```suggestion
```